qiskit/aqua/components/neural_networks/discriminative_network.py

```diff
@@ -34,9 +34,7 @@ def __init__(self) -> None:
         self._num_parameters = 0
         self._num_qubits = 0
         self._bounds = list()  # type: List[object]
-        warn_package('aqua.components.neural_networks',
-                     'qiskit_machine_learning.neural_networks',
-                     'qiskit-machine-learning')
+        warn_package('aqua.components.neural_networks')
```

Comment:
What were here are now in the qgan folder under distribution_learners in machine-learning